tools/checkCasing.js

```diff
@@ -28,8 +28,13 @@ function checkImports(imports) {
         i.resolved = path.resolve(path.dirname(i.path), i.relative + ".ts");
         return i;
     }).filter(function (i) {
-        var entries = fs.readdirSync(path.dirname(i.resolved));
-        return entries.indexOf(path.basename(i.resolved)) === -1;
+        var dirpath = path.dirname(i.resolved);
+        try {
+            var entries = fs.readdirSync(dirpath);
+            return entries.indexOf(path.basename(i.resolved)) === -1;
+        } catch (exception) {
+            console.log("Missing folder for import in " + i.path + ": " + dirpath);
```

Comment:
This needs to set `process.exitCode` as well, or otherwise make sure to return true so it gets picked up by the next stage.